### ISSUE TITLE
hec_enableSSL in default.yml is not working

### ISSUE THREAD

**Issue by BarrieShieh** [splunk/docker-splunk#66]
I set hec_enableSSL to 0 in default.yml to just use http connection for this server, but when I started up the application, the option Enable SSL in Global Settings of HTTP Event Collector is still enabled. Is this parameter not support in the standalone version of 7.2.1?

default.yml is like this

```
---
retry_num: 50
splunk:
  app_paths:
    default: /opt/splunk/etc/apps
    httpinput: /opt/splunk/etc/apps/splunk_httpinput
    idxc: /opt/splunk/etc/master-apps
    shc: /opt/splunk/etc/shcluster/apps
  exec: /opt/splunk/bin/splunk
  group: splunk
  hec_disabled: 0
  hec_enableSSL: 0
  hec_port: 8088
  hec_token: 481673f6-444f-424c-a59e-104015849982
  home: /opt/splunk
  http_port: 8000
  idxc:
    enable: false
    label: idxc_label
    replication_factor: 3
    replication_port: 4001
    search_factor: 3
    secret: 4fCiSGT1o+w7CwiysORYgtTMHlzZVo5z
  opt: /opt
  password: my-secret-pw
  pid: /opt/splunk/var/run/splunk/splunkd.pid
  s2s_port: 9997
  shc:
    enable: false
    label: shc_label
    replication_factor: 3
    replication_port: 4001
    secret: xYLrWSz5YnGa+zwWGUrqDXOcyD828TF9
  svc_port: 8089
  user: splunk
```

**Comment by BarrieShieh** [comment]
Update: I think this issue have been fixed in version 7.2.3.

**Comment by nwang92** [comment]
I'll add a test for this later, but just verified with a defaults.yml with the following set:
```
  hec_disabled: 0
  hec_enableSSL: 0
```

After the container starts, I'm able to use http:
```
ansible@2e52fd983ec7:/$ curl http://localhost:8088/services/collector/event -H "Authorization: Splunk 227d93ad-e5d6-44f2-9512-c039e564fcad" -d '{"event": "hi"}'
{"text":"Success","code":0}
```
